language-support/ts/daml-ledger/index.ts

```diff
@@ -150,6 +150,34 @@ export function assert(b: boolean, m: string): void {
 export type Query<T> = T extends object ? {[K in keyof T]?: Query<T[K]>} : T;
 // TODO(MH): Support comparison queries.
 
+/** @internal
```

Comment:
Did you intend to keep this? It’s just `identity` on `query` with tons of warnings and `console.log`.

Reply:
Not really, I had completely forgotten about it. I remember having trouble silencing the unused warning; the reason this function exists was to remind me that something has to be done about encoding queries at some point. I'll see if I can clean that up a little, and otherwise I'll turn that into a comment instead.